prymer/primer3/primer3.py

```diff
@@ -244,8 +253,20 @@ def __init__(
             executable="primer3_core" if executable is None else executable
         )
         command: list[str] = [f"{executable_path}"]
+        if vcfs_or_lookup is None or isinstance(vcfs_or_lookup, VariantLookup):
+            self.vcfs_or_lookup = vcfs_or_lookup
+        elif isinstance(vcfs_or_lookup, list) and all(
+            isinstance(path, Path) for path in vcfs_or_lookup
+        ):  # if provided path, create appropriate `VariantLookup`
+            self.vcfs_or_lookup = VariantLookup.build(
```

Comment:
This is confusing to me - this class isn't a dataclass or similar.  There's no need to name the member property `self.vcfs_or_lookup` to match the init parameter.  Why not just call it `self.variant_lookup`?